ms_game_chat/backend/backend_app/views.py

```diff
@@ -32,6 +32,7 @@ def createUser(request, user_id):
         new_user.user_id = user_id
         new_user.name = username
         new_user.avatar = avatar
+        new_user.age = 69
```

Comment:
@mariekart42  its just temporal, you should delete the field in your model and remove this line

Reply:
it first needs to get removed in frontend, otherwise there will be errors